res/css/views/dialogs/_InviteDialog.scss

```diff
@@ -316,3 +332,33 @@ limitations under the License.
 .mx_InviteDialog_helpText .mx_AccessibleButton_kind_link {
     padding: 0;
 }
+
+.mx_InviteDialog_dialPad .mx_InviteDialog_dialPadField {
+    border-top: 0px;
+    border-left: 0px;
+    border-right: 0px;
+    border-radius: 0px;
```

Comment:
Probably out of scope for this review, but we might want to introduce a StyleLint rule that forces unit-less `0`.
It's what I usually see in CSS codebases

Reply:
oh yeah - have changed these, but yep, we could probably add a rule